Assets/SEE/UI/RuntimeConfigMenu/RuntimeTabMenu.cs

```diff
@@ -1182,35 +1251,49 @@ private void CreateFilePicker(string settingName, DataPath dataPath, GameObject
             };
         }
 
+        /// <summary>
+        /// Declares a delegate that creates a new instance of a type.
+        /// </summary>
+        /// <typeparam name="T">type to be created</typeparam>
+        /// <returns>creates a new instance of <typeparamref name="T"/></returns>
+        delegate T NewT<T>();
+
         /// <summary>
         /// Creates a list widget.
         /// </summary>
-        /// <param name="list">list to create widget</param>
+        /// <param name="list">list to create a widget for</param>
         /// <param name="parent">parent (container game object: <see cref="CreateNestedSetting"/>)</param>
-        private void CreateList(IList<string> list, GameObject parent)
+        /// <param name="newT">creates a new instance of <typeparamref name="T"/></param>
+        /// <typeparam name="T">the type of elements in <paramref name="list"/></typeparam>
+        private void CreateList<T>(IList<T> list, GameObject parent, NewT<T> newT) where T : class
```

Comment:
Note that this code can be simplified by using `Func<T>` instead of the `NewT<T>` delegate (after which `NewT` is no longer necessary to keep).

An even cleaner solution (though I do not know if it applies here):
The parameter `newT` can possibly be removed entirely and the constraint `where T: class, new()` can be added instead. Afterward, `new T()` can be used (calling the public parameterless constructor of `T`). This also works for, e.g., strings, where the public parameterless constructor returns an empty string.

Reply:
I am now using the suggestion of `Func<T>` now. It may be useful to create an instance with a function that is not the `new` operator.